code/__DEFINES/preferences.dm

```diff
@@ -52,8 +52,10 @@
 #define PREFTOGGLE_2_RUNECHAT		64
 #define PREFTOGGLE_2_DEATHMESSAGE	128
 #define PREFTOGGLE_2_EMOTE_BUBBLE	256
+// Yes I know this being an "enable to disable" is misleading, but it avoids having to tweak all existing pref entries
+#define PREFTOGGLE_2_REVERB_DISABLE	512
```

Comment:
Wouldn't making 1 simple SQL script fix this. And just adding it to the `TOGGLES_2_DEFAULT` value.
Would remove any confusion regarding this define as well. 
A simple `UPDATE preferences_table SET flag=flag+512;` would do (ofc with the correct name etc)

Reply:
I could do that the same as I did for runechat, this is just less error prone

Reply:
Id say this is more error prone due to the double negative you have to do.
Is reverb not not enabled?
If so do reverb